src/server/db.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+
+ const connect = () => {
+  return 	 mongoose.connect(
+		  'mongodb+srv://zoriana:123123321@hackathon2020-673vc.mongodb.net/hack?retryWrites=true&w=majority',
```

Comment:
I think I also need to create the db and set up stuff on my computer. Can you add some info to the README how you did that?

Reply:
no, you dont need. We use Mongo Cloud that I set up.